awx/main/tasks/host_indirect.py

```diff
@@ -77,7 +77,14 @@ def build_indirect_host_data(job: Job, job_event_queries: dict[str, dict[str, st
         if jq_str_for_event not in compiled_jq_expressions:
             compiled_jq_expressions[resolved_action] = jq.compile(jq_str_for_event)
         compiled_jq = compiled_jq_expressions[resolved_action]
-        for data in compiled_jq.input(event.event_data['res']).all():
+
+        try:
+            data_source = compiled_jq.input(event.event_data['res']).all()
+        except Exception as e:
+            logger.error(f'error for module {resolved_action} and data {event.event_data["res"]}: {e}')
```

Comment:
maybe logger.warning might be enough, given it's error in the content ?

Reply:
Sure, I'll change that.